embedding_transfer_learning/run.py

```diff
@@ -1,61 +1,91 @@
-from art.checks import CheckResultExists, CheckScoreExists, CheckScoreCloseTo, CheckScoreLessThan, CheckScoreGreaterThan
-from art.steps import EvaluateBaseline, CheckLossOnInit, OverfitOneBatch, Overfit, Regularize
-from art.loggers import TensorBoardLoggerAdapter
+from dataset import EmbeddingDataModule
 from losses import ApproximateMRR
 from metrics import HitAtKMetric, MRRMetric
+from models.base_model import EmbeddingBaseline, EmbeddingHead
 from steps import DataPreparation, TextDataAnalysis
-from models.base_model import EmbeddingHead, EmbeddingBaseline
-from dataset import EmbeddingDataModule
 
+from art.checks import (
+    CheckResultExists,
+    CheckScoreCloseTo,
+    CheckScoreExists,
+    CheckScoreGreaterThan,
+    CheckScoreLessThan,
+)
+from art.loggers import TensorBoardLoggerAdapter
 from art.project import ArtProject
+from art.steps import (
+    CheckLossOnInit,
+    EvaluateBaseline,
+    Overfit,
+    OverfitOneBatch,
+    Regularize,
+)
 
 
 def getLogger(run_name: str):
     return TensorBoardLoggerAdapter(name=run_name)
 
+
 def main():
     project_name = "embedding_transfer_learning"
-    data_module = EmbeddingDataModule(batch_size=256)
+    data_module = EmbeddingDataModule(batch_size=256, num_workers=0)
     project = ArtProject(project_name, data_module)
-    project.add_step(DataPreparation(),[])
-    project.add_step(TextDataAnalysis(),[
-        CheckResultExists("embedding_size"),
-        CheckResultExists("number_of_reviews"),
-        CheckResultExists("average_review_length"),
-        CheckResultExists("average_query_length"),
-    ])
+    project.add_step(DataPreparation(), [])
+    # project.add_step(
```

Comment:
czm zakomentowane :(

Reply:
dataset nie zwraca teraz tych calych tekstow. Zmienie to w sumie

Reply:
A no i w sumie, Teraz iterujemy sie po wszystkich query wiec trzeba tez bedzie ciut zmienic